libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -870,65 +880,6 @@ def destroy_setup(self, master_node_obj: LogicalNode, worker_node_obj: list,
         except BaseException as error:
             return False, error
 
-    @staticmethod
-    # pylint: disable-msg=too-many-locals
-    def configure_metallb(node_obj: LogicalNode, data_ip: list, control_ip: list):
```

Comment:
why to remove? can we rather shift this to some other folder like as a backup code? ASking just in case we start using LB as ext service and that needs this.

Reply:
will create one backup.